RetroBar/Languages/català.xaml

```diff
@@ -49,13 +110,29 @@
     <s:String x:Key="quick_launch">_Inici ràpid</s:String>
     <s:String x:Key="new_toolbar">Barra d'eines _nova...</s:String>
     <s:String x:Key="set_time">_Ajusta la data i l'hora</s:String>
+    <s:String x:Key="cascade">Finestres en ca_scada</s:String>
+    <s:String x:Key="horiz_tile">Finestres en mosaic _horitzontal</s:String>
+    <s:String x:Key="vert_tile">Finestres en mosaic v_ertical</s:String>
+    <s:String x:Key="toggle_desktop">_Mostra l'escriptori</s:String>
+    <s:String x:Key="hide_desktop">_Mostra les finestres obertes</s:String>
+    <s:String x:Key="undo_cascade">_Desfés En cascada</s:String>
+    <s:String x:Key="undo_tile">_Desfés En mosaic</s:String>
+    <s:String x:Key="undo_minimize_all">_Desfés Minimitza-ho tot</s:String>
     <s:String x:Key="show_taskman_2k">Administrador de _tasques...</s:String>
     <s:String x:Key="show_taskman">Administrador de _tasques</s:String>
     <s:String x:Key="tray_properties">_Propietats</s:String>
     <s:String x:Key="update_available">_Actualització disponible...</s:String>
     <s:String x:Key="exit_retrobar">_Surt del RetroBar</s:String>
     <s:String x:Key="customize_notifications_menu">Personalit_zació de notificacions...</s:String>
 
+    <s:String x:Key="lock_taskbar_menu">_Bloca la barra de tasques</s:String>
+
+    <s:String x:Key="show_desktop_tip_95">Surface/Restore Desktop</s:String>
```

Comment:
Since there isn't an official translation of this, I'll leave it as it is for now.